### PR TITLE
63_send_new_place_on_moderation_Bobeshko

### PR DESCRIPTION
**Pull request by marrybbb**

### CODE REVIEW

**Review by victorpoliushko — APPROVED** [review]

File: src/components/profile/add-place/add-place.js [hunk 1]
```diff
@@ -1,36 +1,106 @@
...
+      title: placeInfo.title,
+      description: placeInfo.description,
+      regionId: (() => {
+        let region;
+        placeInfo.regionsList.find(city => {
+          if (city.name.trim() === placeInfo.region.trim()) {
+            return region = city
+          }
+        }
+        )
```
Comment (innomaxx) [inline src/components/profile/add-place/add-place.js]: Suggestion:
```
const region = placeInfo.regionsList.find(city => city.name.trim() === placeInfo.region.trim());
```

**Comment by innomaxx** [comment]
On page load
image

File: src/components/profile/add-place/add-place.js (hunk 1 above)
Comment (marrybbb) [reply src/components/profile/add-place/add-place.js]: you cant check this task without request on server, unfortunatelly I cant create it till Dima`s request is opened

**Review by victorpoliushko — APPROVED** [review]

File: src/components/profile/add-place/add-place.js [hunk 2]
```diff
@@ -1,36 +1,118 @@
...
+            name="description"
+            value={this.state.description}
+            onChange={this.handleChange}
+            cols="30"
+            rows="10"
           ></textarea>
           <div className="add-place-file-submit">
-            <input className="add-place-file" type="file" />
-            <input className="add-place-submit" type='submit' />
+            <input className="add-place-file"
```
Comment (Viktor-Hordynskyi) [inline src/components/profile/add-place/add-place.js]: pls, add an attribute "required"

File: src/components/profile/add-place/add-place.js [hunk 3]
```diff
@@ -1,36 +1,118 @@
...
-            <option>Rivnenskyi</option>
-            <option>Kyivskyi</option>
-            <option>Lvivskyi</option>
-            <option>Chernigivskyi</option>
+          <select className="district-selector"
+            name="region"
+            required
+            value={this.state.region}
+            onChange={this.handleChange}>
+            <option selected>Choose a region</option>
```
Comment (Viktor-Hordynskyi) [inline src/components/profile/add-place/add-place.js]: add this data - value=""

File: src/components/profile/add-place/add-place.js [hunk 4]
```diff
@@ -1,36 +1,118 @@
...
+  fileSelectedHandler = event => {
+    this.setState({
+      selectedPhotos: event.target.files
+    })
+  }
+
   render() {
+    const { regionsList } = this.state;
+    const elements = regionsList.map((item) => {
+      return (<option key={item._id}>
```
Comment (Viktor-Hordynskyi) [inline src/components/profile/add-place/add-place.js:77]: add this data "value={item.name}"

Comment (marrybbb) [reply src/components/profile/add-place/add-place.js]: everithing is correct here

File: src/components/profile/add-place/add-place.js (hunk 3 above)
Comment (marrybbb) [reply src/components/profile/add-place/add-place.js]: everithing is correct here

File: src/components/profile/add-place/add-place.js (hunk 2 above)
Comment (marrybbb) [reply src/components/profile/add-place/add-place.js]: everithing is correct here

File: src/components/profile/add-place/add-place.js (hunk 4 above)
Comment (marrybbb) [reply src/components/profile/add-place/add-place.js:77]: your comments dont make any sense

**Review by Viktor-Hordynskyi — APPROVED** [review]

**Review by innomaxx — APPROVED** [review]

**Pull request merged**